stacks/linkerd2/deploy.sh

```diff
@@ -1,38 +0,0 @@
-#!/bin/sh
```

Comment:
Did you mean to delete this?

Reply:
@kimschles wanted to ping once more on this - please let us know re: Paul's comment. Thanks!

Reply:
Sorry for the delay. That deletion was a mistake that I fixed in a new commit. Thanks for checking!